hera_mc/cm_utils.py

```diff
@@ -126,9 +120,42 @@ def add_date_time_args(parser):
         '--time', help="UTC hh:mm or float (hours), must include --date if use --time", default=0.0)
 
 
+def is_active(at_date, start_date, stop_date):
+    stop_date = get_stopdate(stop_date)
+    return at_date >= start_date and at_date <= stop_date
+
+
+def future_date():
+    """
+    Future is defined here, since defining a far FUTURE_DATE typically gives a
+    warning about UTC vs UT1 etc
+    """
+    return Time.now() + TimeDelta(300, format='jd')
+
+
+def get_stopdate(stop_date):
+    if isinstance(stop_date, Time):
+        return stop_date
+    else:
+        return future_date()
+
+
+def get_displayTime(display):
```

Comment:
The naming of this function is odd. `get_display_time` instead? Even that I don't think is super transparent, though. `format_time_for_printing`?